daemon/daemon.go

```diff
@@ -913,7 +913,8 @@ func NewDaemonFromDirectory(config *Config, eng *engine.Engine) (*Daemon, error)
 		selinuxSetDisabled()
 	}
 
-	daemonRepo := path.Join(config.Root, "containers")
+	const runDir = "/var/run/docker"
+	daemonRepo := path.Join(runDir, "containers")
```

Comment:
problem was on lxc it was frekaing out about not being able to find /var/run/docker/containers, so maybe its referencing that somewhere else will look